docs/README.md

```diff
@@ -2,9 +2,6 @@
 
 Welcome to the SDK docs!
 
-
-![cosmonaut reading the cosmos docs in space](./cosmos-docs.jpg)
```

Comment:
why rm diagram?

Reply:
I thought it took a lot of visual space and added friction. When people access docs they want immediate information, quick navigation. Just a nit though